jenkins/scripts/slurm_install.sh

```
@@ -20,8 +34,8 @@ slurm_install_setup() {
             rm -f "$lock_file"
         fi
 
-        retry_command --timeout 1800 bash -c "wget -nv $llmTarfile"
-        tar -zxf $tarName
+        download_llm_artifact "$tarName" "$llmTarfile" || return $?
```

Shellcheck (0.11.0)

[warning] tarName is referenced but not assigned.

(SC2154)